dev-dependencies-constants.js

```diff
@@ -8,7 +8,7 @@ exports.MOCHA_TARGET_VERSION = '6.2.3';
 /** Minimum supported typescript version */
 exports.TSC_MIN_VERSION = '3.2.2';
```

Comment:
Do we still support tsc 3?

Reply:
We [used](https://github.com/microsoft/app-store-vsts-extension/blob/master/package.json#L32) it for some methods such as [createExtension ](https://github.com/microsoft/app-store-vsts-extension/blob/master/make-util.js#L1156), bumped version to 5.1.6. Thanks!